closed/OpenJ9.gmk

```diff
@@ -224,6 +224,15 @@ else
   FEATURE_SED_SCRIPT += $(call SedDisable,opt_jitserver)
 endif
 
+# Adjust OpenJDK MethodHandles enablement flags.
+ifeq (true,$(OPENJ9_ENABLE_OPENJDK_METHODHANDLES))
+  SPEC_SED_SCRIPT += $(call SedEnable,opt_openjdkMethodhandle)
+  SPEC_SED_SCRIPT += $(call SedDisable,opt_methodHandle)
+else # OPENJ9_ENABLE_OPENJDK_METHODHANDLES
+  SPEC_SED_SCRIPT += $(call SedEnable,opt_methodHandle)
+  SPEC_SED_SCRIPT += $(call SedDisable,opt_openjdkMethodhandle)
+endif # OPENJ9_ENABLE_OPENJDK_METHODHANDLES
```

Comment:
The contributions to `SPEC_SED_SCRIPT` for `opt_openjdkMethodhandle` and `opt_methodHandle` are unnecessary because eclipse/openj9#10540 removes those flags from all `*.spec` files.
This must, instead, modify `FEATURE_SED_SCRIPT` because those flags are given values in `core.feature`.

Reply:
Fixed. Does this also apply to JDK8, JDK15 and JDK-next?

Reply:
Yes, all jdk versions must work likewise.
I don't see a pull request for jdk8 yet: is it coming soon?

Reply:
Just created few minutes ago: https://github.com/ibmruntimes/openj9-openjdk-jdk8/pull/443.

Reply:
Changed `SPEC_SED_SCRIPT` to `FEATURE_SED_SCRIPT` in all pull requests.